app/models/Search.java

```diff
@@ -323,8 +329,13 @@ private SearchHits getTotalHits(final SearchResponse response) {
 		return hits;
 	}
 
-	private SearchResponse startInitialResponse() {
-		final QueryBuilder queryBuilder = createQuery();
+	private SearchResponse startInitialResponse(final String scroll) {
+		QueryBuilder queryBuilder = createQuery();
+		if (scroll.matches("\\d{8}")) {
+			final QueryBuilder changedSinceQuery =
+					new LobidResources.ChangedSinceQuery().build(scroll);
+			queryBuilder = boolQuery().must(queryBuilder).must(changedSinceQuery);
+		}
```

Comment:
Done. Though I don' like, in general, the usage of global variables. I like to have the used varibles in the method definition, as parameters. Makes them methods easier to understand at a glance, IMO. But may be a matter of taste.